xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -1532,27 +1532,49 @@ bool CGUIMediaWindow::OnPlayAndQueueMedia(const CFileItemPtr& item, const std::s
     // Remove ZIP, RAR files and folders
     CFileItemList playlist;
     playlist.Copy(*m_vecItems, true);
+
+    CLog::LogF(LOGDEBUG,
+               "m_vecItems (in playlist) before removal of folders/RAR/ZIP and duplicates");
+    for (const auto& p : playlist)
+      CLog::LogF(LOGDEBUG, "Path {}, DynPath {}", p->GetPath(), p->GetDynPath());
+
```

Comment:
Please remove this. While this was useful during implementation, once the filtering of the list works, chances are almost zero that it breaks. So, no point in polluting the log with (potentially) many lines.

Reply:
Done